easybuild/easyblocks/p/python.py

```diff
@@ -90,6 +90,10 @@
             if debug:
                 print("[%(EBPYTHONPREFIXES)s] adding site dir: %%s" %% sitedir)
             site.addsitedir(sitedir)
+
+    base_paths = [p for p in sys.path if p in site.getsitepackages()]
```

Comment:
```suggestion
    base_paths = site.getsitepackages()
```
I think this would always be true, but I don't know for sure. Testing out with just virtualenv I can't reproduce any example where site packages aren't always included in `sys.path`? 
(I don't have a strong opinion on this, I think this just happens on python startup right? So performance wise it wouldn't really matter)

Reply:
Yes this likely true, I just wanted to play safe.
And yes this is only done on each invocation of `python`, so not too often. I also moved it above the loop, so it is even faster due to less packages to handle